src/msix/PAL/XML/msxml6/XmlObject.cpp

```diff
@@ -163,6 +163,19 @@ class MSXMLElement final : public ComClass<MSXMLElement, IXmlElement, IMSXMLElem
         return {};
     }
 
+    std::string GetPrefix() override
+    {
+        ComPtr<IXMLDOMNode> node;
+        ThrowHrIfFailed(m_element->QueryInterface(__uuidof(IXMLDOMNode), reinterpret_cast<void**>(&node)));

```

Comment:
>ThrowHrIfFailed(m_element->QueryInterface(__uuidof(IXMLDOMNode), reinterpret_cast<void**>(&node))); [](start = 8, length = 99)

See ComPtr::From

Reply:
We can't use our ComPtr implementation here. This is a windows interface and our ComPtr uses the struct we defined as equivalent of __uuidof

---
In reply to: [305432440](https://github.com/Microsoft/msix-packaging/pull/179#discussion_r305432440) [](ancestors = 305432440)